orchagent/dash/dashaclgroupmgr.cpp

```diff
@@ -601,15 +595,15 @@ task_process_status DashAclGroupMgr::bind(const string& group_id, const string&
     if (group_it == m_groups_table.end())
     {
         SWSS_LOG_INFO("Failed to bind ACL group %s to ENI %s. ACL group does not exist", group_id.c_str(), eni_id.c_str());
-        return task_need_retry;
+        return task_failed;
```

Comment:
In this case, the retry rescheduling leads to a non-deterministic behavior (the same as for binding an empty group).
For example, for the following config:

1. bind group1 (not created yet) - rescheduled
2. create group1
3. add rule1 to group1
4. add rule2 to group2

The bind can get rescheduled to the very end which will work fine, but it can also get rescheduled to be after **(3)** which will lead to the error since we cannot add the **rule2** into the bound group.